src/lib.rs

```diff
@@ -540,3 +542,42 @@ impl Default for WindowAttributes {
         }
     }
 }
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+pub struct VideoMode {
+    dimensions: (u32, u32),
+    bit_depth: u16,
+    frequency: u16,
```

Comment:
you mean refresh rate? Frequency is a pretty vague name.

Reply:
I'm open to bikeshedding this name, but I've commonly seen frequency used as a synonym for the display refresh rate. It is measured in hertz after all, which indicate a frequency..

Reply:
Yeah, but frequency could be frequency of anything. Sound is also measured in hertz. I'd be okay with something like `refresh_frequency` or something.

Reply:
I see. I don't think I've seen it called `refresh_frequency` before. `refresh_rate` is pretty self-explainatory and would work for me.